builder/deltapacks.go

```diff
@@ -75,7 +75,7 @@ func createDeltaPacks(fromMoM *swupd.Manifest, toMoM *swupd.Manifest, printRepor
 			defer wg.Done()
 			for b := range bundleQueue {
 				fmt.Printf("  Creating delta pack for bundle %q from %d to %d\n", b.Name, b.FromVersion, b.ToVersion)
-				info, err := swupd.CreatePack(b.Name, b.FromVersion, b.ToVersion, outputDir, bundleDir, numWorkers)
+				info, err := swupd.CreatePack(b.Name, b.FromVersion, b.ToVersion, outputDir, bundleDir, numWorkers, ignoreMissing)
 				if err != nil {
 					log.Printf("ERROR: Pack %q from %d to %d FAILED to be created: %s\n", b.Name, b.FromVersion, b.ToVersion, err)
 					// Do not exit on errors, we have logging for all other failures and deltas are optional
```

Comment:
Also, I just realize from this comment that ignoring the error is the default behavior. Apparently it wasn't working as such because of that bug you fixed last week. 
Since the errors are logged and ignored, it seems the flag may not even be needed. What we need are tests to check if the behavior is correct and there aren't other panic() calls that are preventing it from continuing on failure.